src/main/java/com/three_tech_solutions/slot_app/controllers/responses/PaymentDetailsResponse.java

```diff
@@ -1,19 +1,11 @@
 package com.three_tech_solutions.slot_app.controllers.responses;
 
-import com.three_tech_solutions.slot_app.data.enums.MonthlyFeeStatus;
-import lombok.AllArgsConstructor;
-import lombok.Data;
+import java.time.LocalDate;
 
-import java.time.LocalDateTime;
-import java.util.UUID;
+public record PaymentDetailsResponse (
+    int monthlyFeeNumber,
+    int paymentNumber,
+    double amount,
+    LocalDate paymentDate
```

Comment:
A esto le podes agregar el @JsonFormat(pattern = "dd/MM/yyyy") para que las fechas no te las devuelva como un array de numeros sino como un string con el formato de la fecha, de esta manera en vez de devolverte la respuesta asi

```
{
    "monthlyFeeNumber": 1,
    "paymentNumber": 1,
    "amount": 27000.0,
    "paymentDate": [
        2025,
        11,
        10
    ]
}
```

te responde asi

```
{
    "monthlyFeeNumber": 1,
    "paymentNumber": 1,
    "amount": 27000.0,
    "paymentDate": "10/11/2025"
}
```



```suggestion
    @JsonFormat(pattern = "dd/MM/yyyy")
    LocalDate paymentDate
```